atd-etl/cris_import/cris_import.py

```diff
@@ -83,21 +87,25 @@ def main():
     GRAPHQL_ENDPOINT_KEY = secrets["graphql_endpoint_key"]
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
+    S3_EXTRACT_BUCKET = secrets["s3_extract_bucket"]
```

Comment:
how do you feel about using the existing vision zero database bucket instead of create a new bucket for this task? the old bucket looks a bit crufty, but i think we could safely purge past extracts from it.

Reply:
We spoke about this, and I believe we settled on using this new bucket on account of having to dole out a credential to access it to another agency and keeping a maximally limited scope on that permission.